Description: Remove Qt4-Webkit dependency
Author: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/784464
Last-Update: 2017-09-06

* asturm: tidied up and amended to use and not override BUILD_WEB

--- a/src/Gui/DownloadManager.cpp
+++ b/src/Gui/DownloadManager.cpp
@@ -33,7 +33,6 @@
 #include <QMetaEnum>
 #include <QSettings>
 #include <QFileIconProvider>
-#include <QWebSettings>
 
 #include "DownloadItem.h"
 #include "DownloadManager.h"
@@ -224,11 +223,12 @@ void DownloadManager::updateRow()
     ui->downloadsView->setRowHeight(row, item->minimumSizeHint().height());
 
     bool remove = false;
+/*
     QWebSettings *globalSettings = QWebSettings::globalSettings();
     if (!item->downloading()
         && globalSettings->testAttribute(QWebSettings::PrivateBrowsingEnabled))
         remove = true;
-
+*/
     if (item->downloadedSuccessfully()
         && removePolicy() == DownloadManager::SuccessFullDownload) {
         remove = true;
--- a/CMakeLists.txt	2018-03-02 22:39:46.374140506 +0100
+++ b/CMakeLists.txt	2018-03-02 22:41:59.904216932 +0100
@@ -351,11 +351,6 @@
   set(BUILD_MESH ON)
 endif(BUILD_JTREADER)
 
-#a runtime dependency
-if(BUILD_START)
-  set(BUILD_WEB ON)
-endif(BUILD_START)
-
 #leaving test module for someone else.
 #image has none. as far as I can tell.
 #mesh has none. as far as I can tell.
